NewportSMC100/dialogSMC100.cpp

```diff
@@ -43,7 +43,7 @@ DialogSMC100::DialogSMC100(ito::AddInBase *actuator) :
     m_calibStatusNames.append(tr("MZ and encoder"));
     m_calibStatusNames.append(tr("Current Pos as Home"));
     m_calibStatusNames.append(tr("MZ only"));
-    m_calibStatusNames.append(tr("EoR ans encoder"));
+    m_calibStatusNames.append(tr("EoR and encoder"));
```

Comment:
is **and** the right replacement? I don't know

Reply:
line 43 it is "MZ and encoder" so I assumes the same for line 46